### PR TITLE
Log dbench

### PR DESCRIPTION
**Pull request by nigelbabu**
Remove the non-working archival bits.

### CODE REVIEW

File: smoke.sh [hunk 1]
```diff
@@ -62,10 +63,7 @@ function finish ()
 {
     RET=$?
     if [ $RET -ne 0 ]; then
-        mkdir -p /d/logs/smoke
-        filename="/d/logs/smoke/glusterfs-logs-`date +%Y%m%d%H%M%S`.tgz"
-        tar -czf $filename /build/install/var/log;
-        echo Logs archived in $filename
```
Comment (kshlm) [inline smoke.sh]: Why has this been removed? Capturing the GlusterFS logs could be useful.

File: smoke.sh [hunk 2]
```diff
@@ -14,6 +14,7 @@ function cleanup()
     killall -15 glusterfs glusterfsd glusterd 2>&1 || true;
     killall -9 glusterfs glusterfsd glusterd 2>&1 || true;
     umount -l $M 2>&1 || true;
+    rm /build/dbench
```
Comment (kshlm) [inline smoke.sh]: s/dbench/dbench-logs/

File: smoke.sh (hunk 1 above)
Comment (nigelbabu) [inline smoke.sh]: There's nothing in `/build/install/var/log;` on any machine. So we've most likely gotten the path wrong. Right now it's not doing anything useful (except maybe throwing a false flag)

**Comment by kshlm** [comment]
LGTM :+1:

**Pull request merged**